datarockets-style.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "rubocop", "~> 1.45.1"
+  spec.add_dependency "rubocop-graphql", "~> 1.1.1"
```

Comment:
Can not use later versions because they depend on a later version of rubocop

Reply:
@Anastastasia-B I'll upgrade rubocop before pushing the new version, so don't worry 😉

Reply:
Turned out that versions after 1.1.1 also depend on Ruby 3.0 so still can not use later versions